src/vs/workbench/contrib/terminalContrib/find/browser/terminal.find.contribution.ts

```diff
@@ -134,7 +134,7 @@ registerActiveXtermAction({
 	keybinding: {
 		primary: KeyMod.Alt | KeyCode.KeyR,
 		mac: { primary: KeyMod.CtrlCmd | KeyMod.Alt | KeyCode.KeyR },
-		when: ContextKeyExpr.or(TerminalContextKeys.focusInAny, TerminalContextKeys.findFocus),
+		when: TerminalContextKeys.findVisible,
```

Comment:
@Tyriar shouldn't this be `findFocus`

Reply:
That's what I did originally but I thought visible makes more sense